### ISSUE TITLE
Update Munich descendants data

### ISSUE THREAD

**Issue by CeallachKN** [whosonfirst-data/whosonfirst-data#2168]
Hey there

TLDR:
The descendants for Munich are really out of date, mostly is_current=-1 and the geometrys nowhere close to being at the right place. They should be updated with official data.

![grafik](https://github.com/whosonfirst-data/whosonfirst-data/assets/68635133/779b9636-e4e0-48a2-9477-6e610074977c)
Polygon:
Green = WOF Munich Locality, Localadmin, County
Red = WOF Munich descendant neighbourhood with mz:is_current=0
Blue =  WOF Munich descendant neighbourhood with mz:is_current=1

Point:
All Point-Features with placetype localadmin have been deprecated
placetype locality: 2 have been deprecated, one is only is_current = 0 (Red Diamond)
placetype neighbourhood: 87 are is_current = -1 (Blue Diamond), 1 is is_current= 1 (Green Diamond) and 3 are deprecated  

Munich makes data available at https://opendata.muenchen.de/organization/geodatenservice-muenchen mostly with a https://www.govdata.de/dl-de/by-2-0 License (Terms of use: https://opendata.muenchen.de/pages/nutzungsbedingungen)

The geometries are divided by the following hierarchical levels:
25 Stadtbezirke (Borough / Macrohood ?) : Have a name and a Reference Number (e.g. 19) 
--> Data: https://opendata.muenchen.de/dataset/vablock_stadtbezirke_opendata
--> Wikipedia: https://de.wikipedia.org/wiki/Stadtbezirke_M%C3%BCnchens
108 Bezirksteile (Neighbourhoods): Only a Reference Number (e.g. 19.1)
--> https://opendata.muenchen.de/dataset/vablock_bezirksteil_opendata
--> Wikipedia: Referenced in https://de.wikipedia.org/wiki/Stadtbezirke_M%C3%BCnchens in the table in "Weitere Unterteilung" or in this PNG https://de.wikipedia.org/wiki/Liste_der_Stadtteile_M%C3%BCnchens#/media/Datei:Karte_der_Stadtbezirksteile_in_M%C3%BCnchen.png
475 Stadtbezirksviertel (???):  Only a Reference Number (e.g. 19.1.1)
--> https://opendata.muenchen.de/dataset/vablock_viertel_opendata 
--> According to Wikipedia only 237 of 475 of these have names (not in the data), but the only reference for these names and proposed names for the rest (according to the article derived from historical names aka "how the citziens probalby would call it") i found was in this appendix to the wikipedia-Article

![grafik](https://github.com/whosonfirst-data/whosonfirst-data/assets/68635133/80aee9cf-9f88-433a-ac36-9d9df83a0c02)
![grafik](https://github.com/whosonfirst-data/whosonfirst-data/assets/68635133/71343149-b9a7-4b42-8dd6-20877074e2a3)
![grafik](https://github.com/whosonfirst-data/whosonfirst-data/assets/68635133/59d60456-ef6c-4729-bc34-6e14c46053cc)

But to be clear, the geometry data is administrative for "Stadtbezirke", as these have some kind of adimistrative control and mostly statistical for "Bezirksteile" and "Stadtviertel". 
- Most "Stadtbezirksviertel" have a historical Name, and the names for "Stadtbezirke" and "Bezirktsteile" are made up of these historical names, but sadly the names aren't used for "Bezirksteile" in the data provided by Open Data Muenchen. In Wikipedia / Wikidata the are some references for example https://www.wikidata.org/wiki/Q971198 , https://de.wikipedia.org/wiki/Altstadt-Lehel
- Locals usually use a lot of (historical) names that aren't necessarily geometrically identical to the data provided above as well. Referenced is this in the PNG in the Wikipedia-Article https://de.wikipedia.org/wiki/Stadtbezirke_M%C3%BCnchens#/media/Datei:Karte_der_Ortsbezeichnungen_in_M%C3%BCnchen.png and in https://de.wikipedia.org/wiki/Liste_der_Stadtteile_M%C3%BCnchens#
- Additional population data https://opendata.muenchen.de/dataset/bevoelkerung-stadtbezirken for "Stadtbezirke" and https://opendata.muenchen.de/dataset/bevoelkerung-stadtbezirksteile-muenchen for "Stadtbezirksteile" from Open Data Muenchen could be used as well. 

My guess is that this would warrant a whole new set of records, with a combination of data from Open Data Muenchen, Wikidata / Wikipedia, existing WOF data etc. to make sense.
I would be happy to at least see the "Stadtbezirke" updated, the rest would need a lot of data grinding to make sense, since they don't even have an official name in the dataset itself.

As additional info: The issue with gaps and overlayed geometrys is the same as mentioned in https://github.com/whosonfirst-data/whosonfirst-data-admin-de/pull/111#issuecomment-1750841728 , but since it is contained in Bavaria i guess in this case it should be solved?
As i see it the ldbv-data in https://github.com/whosonfirst-data/whosonfirst-data-admin-de/pull/111 is less accurate then the dataset linked here, when compared with satellite-basemap or osm-basemap. Relevant for Pelias-Adress-Search / Reverse-Geocoding since Pelias relies heavily on wof-lookup even for complete Adress-data. 

Some Screenshots with OSM as basemap
brown line: WOF
pink line: ldbv-data
blue line:  Open Data Muenchen ("Stadtbezirke" in these cases)
![grafik](https://github.com/whosonfirst-data/whosonfirst-data/assets/68635133/f27e9762-4816-4e6a-9a2c-25127f9c5324)
![grafik](https://github.com/whosonfirst-data/whosonfirst-data/assets/68635133/dceca993-3b45-4925-91a6-62bf22d9fb5b)
![grafik](https://github.com/whosonfirst-data/whosonfirst-data/assets/68635133/faa131b5-d541-422f-81b8-692cc525baea)

**Comment by nvkelso** [comment]
Thanks for the detailed report! I agree the data could be better here, and your source and proposal looks promising! When I free up from edits in United Kingdom and this week's conference I'll have a deeper look and respond to your points...

**Comment by CeallachKN** [comment]
@nvkelso: Have you had time to look at this and [#111](https://github.com/whosonfirst-data/whosonfirst-data-admin-de/pull/111) ? Can we (@arnesetzer as well) / I help in some way?

Would [#111 ](https://github.com/whosonfirst-data/whosonfirst-data-admin-de/pull/111) need to be finalized and merged first?

**Comment by CeallachKN** [comment]
I've just find this population-dataset for the "Stadtbezirksteile" (Neighbourhoods) https://opendata.muenchen.de/dataset/bevoelkerung-stadtbezirksteile-muenchen that also contains the official names linked to the reference numbers. 

I could try to author a dataset combining the existing wof-data with the linked datasets. My only question is if i should generate new ids for all of them (even the locality/localadmin for Munich https://spelunker.whosonfirst.org/id/101748479/ ) or reuse existing IDs (at least for the locality / localadmin).
I would try to keep as much of the existing information as possible, especially for names in other languages, as long as they faven't been incorrect all along.

But this would take me a while since my workload hasn't lessened.
